wallet/test/run-wallet.c

```diff
@@ -980,6 +983,8 @@ static bool test_wallet_outputs(struct lightningd *ld, const tal_t *ctx)
 	u.close_info->channel_id = 42;
 	u.close_info->peer_id = id;
 	u.close_info->commitment_point = NULL;
+	u.scriptPubkey = tal_arrz(w, u8, 20);
```

Comment:
Why `talarrz` if we are going to fill the entire array with `1`s later?

Reply:
Trivial fix pushed.

Reply:
Hmmmm does it fix this? confused

Reply:
Ah, your fix only fixes one instance of this, this is another one.

Reply:
Damn, will patch once kids in bed...

Reply:
Fixed :wink: